sdv/multi_table/hma.py

```diff
@@ -9,11 +9,12 @@
 from tqdm import tqdm
 
 from sdv.multi_table.base import BaseMultiTableSynthesizer
+from sdv.sampling import BaseHierarchicalSampler
 
 LOGGER = logging.getLogger(__name__)
 
 
-class HMASynthesizer(BaseMultiTableSynthesizer):
+class HMASynthesizer(BaseHierarchicalSampler, BaseMultiTableSynthesizer):
```

Comment:
Does this order matter? If not could we flip it 😅

Reply:
Order does matter, if we don't have `BaseHierarchicalSampler` first then it'll hit the `BaseMultiTableSynthesizer` methods _first_, so we'll end up hitting `NotImplemented` errors.